core/domain/email_manager.py

```diff
@@ -2358,3 +2360,25 @@ def send_not_mergeable_change_list_to_admin_for_review(
         email_body = email_body_template % (
             exp_id, change_list_dict, frontend_version, backend_version)
         send_mail_to_admin(email_subject, email_body)
+
+
+def verify_mailchimp_secret(secret: str) -> bool:
+    """Verifies the secret key for the mailchimp webhook.
+
+    Args:
+        secret: str. The secret key provided by the mailchimp webhook.
+
+    Returns:
+        bool. Whether the secret key is valid.
+    """
+    mailchimp_webhook_secret = secrets_services.get_secret(
+        'MAILCHIMP_WEBHOOK_SECRET')
+    if mailchimp_webhook_secret is None:
+        logging.exception('Cloud Secret Manager is not working.')
+        if feconf.MAILCHIMP_WEBHOOK_SECRET is None:
+            logging.exception('Mailchimp webhook secret is not available.')
+            return False
+
+        return secret == feconf.MAILCHIMP_WEBHOOK_SECRET
```

Comment:
Why do we need to store the secret in feconf? Shouldn't we always get it from the secrets_services instead?